### PR TITLE
[BALANCE|FIX]ERT Balance Pass: 1 + Minor Ops Ship Fix

### PR DESCRIPTION
**Pull request by Steelpoint**
After some feedback from players this is the first balance pass I will be making, its a minor one though.

Features
- Combat Defibrillator: A scaled down Defib unit for ERT Med Officers, can fit in your backpack. Functions identically to the normal defib. Uses custom sprites.
- Replaced the Energy Guns in the ERT Armoury with Pulse Carbines.
- The Pulse Pistol & Carbine now use Electronic Loyalty Firing Pins, meaning only loyally implanted people can use their guns.
- ERT Officers now have all access to the station.
- Changed Pulse Carbine/Pistols to use a normal cell and not a super cell, this is to get rid of the 10 minutes recharge time. No ammo change.

---
- Fixed the Ops shuttle/base misaligned areas.
- Fixes missing firing pins for ERT Ion Rifles.

### CODE REVIEW

**Comment by Steelpoint** [comment]
@paprka may be interested in the defibrillator.

**Comment by Dorsisdwarf** [comment]
Uh, what?
From my interactions in the playtest on the highlander round, when the ERT have pulse weapons, they will use pulse weapons, and only pulse weapons, for that 50 damage deathshot. It's not like they're going to get in trouble for pulsing people to death instead of lasering them, so why would they ever use anything but DESTROY?

**Comment by Steelpoint** [comment]
Is this merge ready?

**Comment by Steelpoint** [comment]
Doing some additional balance changes based off a recent, and very successful, ERT run.

**Comment by Steelpoint** [comment]
More feedback from the server.

People really seem to like the ERT, more so the admins, and they seem to do well for themselves in any given situation, from combat operations to rescue and repair. There were some additional concerns raised though.

* ERT Officers have a hard time figuring out which role they are.
* Their Guns take way too long to recharge.
* They lack access to anywhere on the station, thus forcing them to use their Pulse guns, thus making point two more obviously worse.

As such I've added two additional changes, firstly all Officers will now have All Access to the entire station (one assume Centcom would be at least that generous). Secondly I've changed the personal lockers to show a departmental sprite corresponding to the ID it holds, so a Med Officers locker will show a Med Locker sprite. I've also better highlighted the Suit Storage units.

**Comment by paprka** [comment]
I think people are just stupid and wait for their guns to fully recharge. They recharge at the exact same rate as any other gun

**Comment by Steelpoint** [comment]
Pulse Guns use a very high cap cell, meaning it takes around 8 minutes to fully recharge any Pulse Gun, Rifle to Pistol. 

I'll investigate Sec Uniforms later on and post some pics on the forums BEFORE I do a PR that results in 200 comments.

**Comment by paprka** [comment]
yeah you can find the code somewhere in cheridan's pin PR if it wasn't added, he might have removed it but it's in the first commit i think

**Comment by Steelpoint** [comment]
This is merge ready, can it be merged?

**Comment by Steelpoint** [comment]
20 minutes for travis to finish....

This is merge ready.

**Comment by paprka** [comment]
sorry we need 6 weeks to review this

**Comment by Steelpoint** [comment]
I fixed the missing firing pins inside the Ion Guns, and also gave all ERT Pulse weapons a loyalty firing pin.

**Comment by Steelpoint** [comment]
Making me beg. Can we please merge this.

File: code/game/objects/items/weapons/cards_ids.dm [hunk 1]
```diff
@@ -196,7 +196,17 @@ update_label("John Doe", "Clowny")
...
+	            access_medical, access_genetics, access_morgue, access_rd,
+	            access_tox, access_tox_storage, access_chemistry, access_engine, access_engine_equip, access_maint_tunnels,
+	            access_external_airlocks, access_change_ids, access_ai_upload,
+	            access_teleporter, access_eva, access_heads, access_captain, access_all_personal_lockers,
+	            access_tech_storage, access_chapel_office, access_atmospherics, access_kitchen,
+	            access_bar, access_janitor, access_crematorium, access_robotics, access_cargo, access_construction,
+	            access_hydroponics, access_library, access_lawyer, access_virology, access_cmo, access_qm, access_surgery,
+	            access_theatre, access_research, access_mining, access_mailsorting, access_weapons,
+	            access_heads_vault, access_mining_station, access_xenobiology, access_ce, access_hop, access_hos, access_RC_announce,
+	            access_keycard_auth, access_tcomsat, access_gateway, access_mineral_storeroom, access_minisat)
```
Comment (Miauw62) [inline code/game/objects/items/weapons/cards_ids.dm]: just make one ERT type and derive the different cards from that, don't copypaste all these accesses for every type.

**Comment by Steelpoint** [comment]
Done, I refined the access.

File: code/game/objects/items/weapons/cards_ids.dm [hunk 2]
```diff
@@ -196,7 +196,8 @@ update_label("John Doe", "Clowny")
 	icon_state = "centcom"
 	registered_name = "Emergency Response Team Commander"
 	assignment = "Emergency Response Team Commander"
-	access = list(access_sec_doors, access_engine, access_medical)
+	New()
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/cards_ids.dm]: Full typepaths please. also ```access = get_all_accesses()``` wasn't quite what you were asked to do.
Miauw meant make an ```/obj/item/weapon/card/id/ert``` which has ```access = get_all_accesses()``` on it (or some other form of all access) and just have ```/obj/item/weapon/card/id/ert/Security```, ```/obj/item/weapon/card/id/ert/Engineer``` etc.

**Comment by Steelpoint** [comment]
Additional refinement

**Comment by Steelpoint** [comment]
Jesus these travis wait times.

File: code/modules/projectiles/guns/energy/pulse.dm [hunk 3]
```diff
@@ -21,6 +21,10 @@
 	icon_state = "pulse_carbine"
 	item_state = "pulse"
 	ammo_type = list(/obj/item/ammo_casing/energy/laser/pulse/carbine, /obj/item/ammo_casing/energy/electrode/carbine, /obj/item/ammo_casing/energy/laser/carbine)
+	cell_type = "/obj/item/weapon/stock_parts/cell"
+
+/obj/item/weapon/gun/energy/pulse_rifle/pulse_carbine/loyalpin
+	pin = "/obj/item/device/firing_pin/implant/loyalty"
```
Comment (RemieRichards) [inline code/modules/projectiles/guns/energy/pulse.dm]: ```/obj/item/device/firing_pin/implant/loyalty``` not ```"/obj/item/device/firing_pin/implant/loyalty"```

File: code/modules/projectiles/guns/energy/pulse.dm [hunk 4]
```diff
@@ -29,6 +33,10 @@
 	icon_state = "pulse_pistol"
 	item_state = "pulse"
 	ammo_type = list(/obj/item/ammo_casing/energy/laser/pulse/pistol, /obj/item/ammo_casing/energy/electrode/pistol, /obj/item/ammo_casing/energy/laser/pistol)
+	cell_type = "/obj/item/weapon/stock_parts/cell"
+
+/obj/item/weapon/gun/energy/pulse_rifle/pulse_pistol/loyalpin
+	pin = "/obj/item/device/firing_pin/implant/loyalty"
```
Comment (RemieRichards) [inline code/modules/projectiles/guns/energy/pulse.dm]: ```/obj/item/device/firing_pin/implant/loyalty``` not ```"/obj/item/device/firing_pin/implant/loyalty"```

**Comment by Steelpoint** [comment]
Travis is, again, finished.

**Comment by Steelpoint** [comment]
This is ready to merge?!?!?

**Pull request merged**